bucky/src/MultiplyTwoNumbers.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Scanner;
+
+public class MultiplyTwoNumbers {
+    public static void main(String[] args) {
+        Scanner multiply = new Scanner(System.in);
+        System.out.println("first number");
+        int firstNumber = multiply.nextInt();
+        System.out.println("second number");
+        int secondNumber = multiply.nextInt();
+        int total = firstNumber * secondNumber;
```

Comment:
change variable name from `total` to `product`

Reply:
done